wiki/PlaceholderExpansion.md

```diff
@@ -58,19 +58,27 @@ public class SomeExpansion extends PlaceholderExpansion {
 Let's quickly break down the different methods you had to import.
 
 - #### getAuthor
-  Through this method can you set the name of who created the expansion.
+  Through this method you can set the name of who created the expansion.
 - #### getIdentifier
   The name that should be used to identify the placeholders for this expansion.  
   The Identifier is the first text after the `%` and before the first `_` (`%identifier_values%`) and can therefore not contain any `_` in it.
```

Comment:
```suggestion
The identifier is the string after the starting `%` and before the first `_` (`%**identifier**_values%`) and, therefore, cannot contain any `_`s.
```

Reply:
** won't really work here since it's in a inline code. Also, I doubt people are THAT dumb to not recognize the part literally called "identifier"

Reply:
Ah okay, can keep the identifier normal then, but I would suggest implementing the rest of the changes!